ods_ci/tasks/Resources/Files/monitoring-patch-payload.json

```diff
@@ -2,6 +2,8 @@
     "spec": {
```

Comment:
why this file change?

Reply:
better to remove from this PR, maybe just try to update the branch from GH directly, let's see if it disappear

Reply:
removed